app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,278 @@
+<p>
+    Appwrite provides authentication for many different use cases to fit the needs of developers. 
+    Appwrite manages authentication with a combination of users and sessions. 
```

Comment:
> users and sessions

Maybe user accounts and sessions or accounts and sessions because the API methods say "accounts" and I believe the new UI coming soon will stick with accounts.